test_scripts/RC/OnRemoteControlSettings/009_Allowed_true_accessMode_AUTO_ALLOW.lua

```diff
@@ -0,0 +1,54 @@
+---------------------------------------------------------------------------------------------------
+-- Requirement summary:
+-- [SDL_RC] Resource allocation based on access mode
+--
+-- Description:
+-- In case:
+-- RC_functionality is disabled on HMI and HMI sends notification OnRemoteControlSettings (allowed:true, <any_accessMode>)
+--
+-- SDL must:
+-- 1) store RC state allowed:true and received from HMI internally
+-- 2) allow RC functionality for applications with REMOTE_CONTROL appHMIType
+--
+-- Case: accessMode = "AUTO_ALLOW"
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local commonRC = require('test_scripts/RC/commonRC')
+
+--[[ Local Variables ]]
+local modules = { "CLIMATE", "RADIO" }
+
+local rcRpcs = { "SetInteriorVehicleData", "ButtonPress" }
+
+--[[ Local Functions ]]
+
+local function ptu_update_func(tbl)
+  tbl.policy_table.app_policies[config.application2.registerAppInterfaceParams.appID] = commonRC.getRCAppConfig()
+end
+
+local function disableRcFromHmi(self)
+  commonRC.defineRAMode(false, nil, self)
+end
+
+--[[ Scenario ]]
+runner.Title("Preconditions")
+runner.Step("Clean environment", commonRC.preconditions)
+runner.Step("Start SDL, HMI, connect Mobile, start Session", commonRC.start)
+runner.Step("RAI1, PTU", commonRC.rai_ptu, { ptu_update_func })
+runner.Step("RAI2", commonRC.rai_n, { 2 })
+runner.Step("Disable RC from HMI", disableRcFromHmi)
+
+runner.Title("Test")
+runner.Step("Enable RC from HMI with AUTO_ALLOW access mode", commonRC.defineRAMode, { true, "AUTO_ALLOW"})
+for _, mod in pairs(modules) do
+  for _, rpc in pairs(rcRpcs) do
+    runner.Step("Activate App1", commonRC.activate_app)
+    runner.Step("Check module " .. mod .." App1 " .. rpc .. " allowed", commonRC.rpcAllowed, { mod, 1, rpc })
+    runner.Step("Activate App2", commonRC.activate_app, { 2 })
+    runner.Step("Check module " .. mod .." App2 " .. rpc .. " allowed", commonRC.rpcAllowed, { mod, 2, rpc })
+  end
+end
+
```

Comment:
@Itileda 
sequence: disable-check_if_disallowed is covered in test_scripts/RC/OnRemoteControlSettings/008_Allowed_false.lua
This check is redundant for current test.